### PR TITLE
33590 alt text for content ff release

### PR DESCRIPTION
**Pull request by dletorey**
### Description

- Added experimental release note for alt text in `content` property
- Added FF128 release note for alt text in `content` property

### Motivation

Working on [issue 33590](https://github.com/mdn/content/issues/33590)

### Related issues and pull requests

- [BCD PR](https://github.com/mdn/browser-compat-data/pull/23199)
- [Content PR](https://github.com/mdn/content/pull/33834)

### CODE REVIEW

File: files/en-us/mozilla/firefox/experimental_features/index.md [hunk 1]
```diff
@@ -184,6 +184,51 @@ The {{cssxref("initial-letter")}} CSS property is part of the [CSS Inline Layout
...
+      <th>Preference name</th>
+      <td colspan="2"><code>layout.css.content.alt-text.enabled</code></td>
+    </tr>
+  </tbody>
+</table>
+
+Note:
+
+- The alternative text is not exposed to the accessibility tree in versions prior to 128 due to [Firefox bug 1859852](https://bugzil.la/1859852).
+- Although, as the alternative text syntax is now parsed, the image added using `content` property is displayed.
```
Comment (pepelsbey) [inline files/en-us/mozilla/firefox/experimental_features/index.md:230]: I wonder if this feature was implemented (not supported before) or just fixed to work properly. The first paragraph gives the first impression. The note gives the second. Now I’m curious :)

**Review by pepelsbey — COMMENTED** [review]
Looks good overall, thanks! Just one question.

File: files/en-us/mozilla/firefox/experimental_features/index.md (hunk 1 above)
Comment (dletorey) [reply files/en-us/mozilla/firefox/experimental_features/index.md:230]: No the feature was not supported before.
There are several parts to this and the first is making sure that the **/** (slash) syntax is not dismissed as bad syntax. The second part is then actually adding the value after the **/** (aka the alt text) into the accessibility tree

**Review by pepelsbey — COMMENTED** [review]
Since this feature is going to be available in 128 behind the flag, I'd also feature it in the [experimental features](https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Releases/128#experimental_web_features) section of the 128 release notes. Featuring experimental features gives developers a chance to test them before we ship them.

**Comment by dletorey** [comment]
> Since this feature is going to be available in 128 behind the flag, I'd also feature it in the [experimental features](https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Releases/128#experimental_web_features) section of the 128 release notes. Featuring experimental features gives developers a chance to test them before we ship them.

This is definitely working in Firefox Beta (128) without the flag cc @pepelsbey

**Review by pepelsbey — APPROVED** [review]
No experiments then, all right :) Thanks!

**Pull request merged**